src/app/static/src/tests/directives/translate.test.ts

```diff
@@ -211,13 +211,17 @@ describe("translate directive", () => {
                 plugins: [store]
             }, 
         });
-        expect((store._watcherVM as any)._watchers.length).toBe(5);
+        expect((renderedAttribute.element as any).__lang_unwatch__).toHaveProperty("value")
+        expect((renderedText.element as any).__lang_unwatch__).toHaveProperty("innerHTML")
+        expect((renderedMultiple.element as any).__lang_unwatch__).toHaveProperty("innerHTML")
+        expect((renderedMultiple.element as any).__lang_unwatch__).toHaveProperty("placeholder")
+        expect((renderedMultiple.element as any).__lang_unwatch__).toHaveProperty("value")
         renderedAttribute.unmount();
-        expect((store._watcherVM as any)._watchers.length).toBe(4);
+        expect((renderedAttribute.element as any).__lang_unwatch__).toStrictEqual({})
         renderedText.unmount();
-        expect((store._watcherVM as any)._watchers.length).toBe(3);
+        expect((renderedText.element as any).__lang_unwatch__).toStrictEqual({})
         renderedMultiple.unmount();
-        expect((store._watcherVM as any)._watchers.length).toBe(0);
+        expect((renderedMultiple.element as any).__lang_unwatch__).toStrictEqual({})
```

Comment:
Wow, this is even more gribbly than before. Can we really not view these values from the store any more?

Reply:
ive tried to explore this a bit more and so far I cannot really see any way of getting watcher info from the store :( do have a go if you have time but I think theyve just change the way they have implemented store watchers because they don't seem to be storing any of it on the store object

Reply:
Fair enough!